visualizations/frontend/other/TTable.vue

```diff
@@ -1345,17 +1342,18 @@ highlighting a row on hover etc. */
 
 .tableHeaderContainer {
   display: flex;
+  flex-wrap: wrap;
+  align-items: center;
   justify-content: space-between;
   width: 100%;
   overflow: visible;
+  gap: 15px;
 }
 .tableControls {
   display: flex;
   gap: 10px;
   align-items: center;
+  justify-content: flex-end;
 }
 
-.csvExportButton{
-  min-width: 130px;
-}
```

Comment:
This min-width resulted in extra mystery space, that was pushing away from the edge of the table

### Before change:
<img width="199" alt="Screenshot 2022-09-29 at 13 29 44" src="https://user-images.githubusercontent.com/1307818/193020389-b1e05883-d5a2-4e9c-b81c-271dfc071510.png">

If we remove this and rely on the `gap` property the effect is quite nice 

### After change:
<img width="1306" alt="Screenshot 2022-09-29 at 13 32 41" src="https://user-images.githubusercontent.com/1307818/193020728-713b0bbc-5d30-4595-9c16-5a83cea1daae.png">
<img width="327" alt="Screenshot 2022-09-29 at 13 32 32" src="https://user-images.githubusercontent.com/1307818/193020733-b10b2114-c226-444c-ad55-5fcc66b8aae9.png">

Reply:
The min-width was added to fix the download CSV button ending up on 2 lines when present with the modify columns button:
<img width="372" alt="image" src="https://user-images.githubusercontent.com/78518576/193297343-8a8c25a0-47d3-47f1-bfc1-4a76c4639732.png">

Replacing min-width in the csvExportButton class with "white-space: nowrap" should fix it though.